extensions/adobe/experience/decisioning/decisionevent.example.1.json

```diff
@@ -0,0 +1,65 @@
+{
+  "@id": "https://data.adobe.io/entities/propositions/3cc33a7e-13ca-4b19-b25d-c816eff9a70a",
+  "xdm:timestamp": "2019-01-26T15:52:25+00:00",
+  "xdm:identityMap": {
+    "ECID": [
+      { "xdm:id": "https://data.adobe.io/entities/identity/49312748749132" }
+    ]
+  },
+  "xdm:device": {
+    "xdm:typeID": "IMac",
+    "xdm:typeIDService": "https://ns.adobe.com/xdm/external/adobecampaign",
+    "xdm:type": "desktop",
+    "xdm:manufacturer": "Apple"
+  },
+  "xdm:placeContext": {
+    "xdm:localTime": "2017-09-26T15:52:25+13:00",
+    "xdm:geo": {
+      "@id": "https://data.adobe.io/entities/geo/tokyo",
+      "xdm:countryCode": "JP",
+      "xdm:stateProvince": "JP-13",
+      "xdm:city": "Tōkyō",
+      "xdm:postalCode": "141-0032",
+      "schema:latitude": 35.6185,
+      "schema:longitude": 139.73237
+    }
+  },
+  "https://ns.adobe.com/experience/decisioning/propositionID": "3cc33a7e-13ca-4b19-b25d-c816eff9a70a",
+  "https://ns.adobe.com/experience/decisioning/propositionDetails": [
+    {
+      "xdm:activity": {
+        "@id": "xcore:offer-activity:ebc48132c26ccfc"
+      },
+      "xdm:selections": [
+        {
+          "@id": "xcore:personalized-offer:e526a1503bf29e9"
+        }
+      ],
+      "xdm:metrics": {
+        "https://ns.adobe.com/experience/offer-management/offer-propositions": {
+          "xdm:value": 948
+        }
+      }
+    },
+    {
+      "xdm:activity": {
+        "@id": "xcore:offer-activity:f203512e02542b5"
+      },
+      "xdm:fallback": {
```

Comment:
Is this not supposed to be inside the "xdm:activity" ?

Reply:
No this is the output of the decision, it indicates that no selection was made and the fallback is being proposed. This is different from the snapshot of the activity that MAY include the fact that the activity had a fallback defined. The snapshot data is purely optional whereas the output of the decision is that either some options were selected for the profile or the fallback option would be the only one that applies